Signal/src/environment/SignalApp.h

```diff
@@ -35,16 +35,24 @@ NS_ASSUME_NONNULL_BEGIN
 
 + (instancetype)sharedApp;
 
-#pragma mark - View Convenience Methods
+#pragma mark - Conversation Presentation
+
+- (void)presentConversationForRecipientId:(NSString *)recipientId animated:(BOOL)isAnimated;
```

Comment:
just added an `animated` flag to each of these, which ultimately gets passed to `setViewControllers:@[self, conversationVC] animated:**isAnimated**` in the HomeViewController